src/Telegram.Bot/Exceptions/ApiException.cs

```diff
@@ -64,10 +66,88 @@ public ApiRequestException(string message, int errorCode, Exception innerExcepti
+                            return new NotEnoughRightsException(apiResponse.Message)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                        case "user not found":
+                            return new UserNotFoundException(apiResponse.Message)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                        case "method is available for supergroup and channel chats only":
+                            return new WrongChatTypeException(apiResponse.Message)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                        default:
+                            if (message.EndsWith(" is empty"))
+                            {
+                                return new MissingParameterException(apiResponse.Message, message.Remove(message.IndexOf(" is empty")))
+                                {
+                                    Parameters = apiResponse.Parameters
+                                };
+                            }
+                            LogMissingError(apiResponse);
+                            return new ApiRequestException(apiResponse.Message, 400)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                    }
+                case 403:
+                    message = apiResponse.Message.Remove(0, "Forbidden: ".Length);
+                    switch (message.Trim())
+                    {
+                        case "bot was blocked by the user":
+                            return new BotBlockedException(apiResponse.Message)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                        case "bot can't initiate conversation with a user":
+                            return new BotNotStartedException(apiResponse.Message)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                        default:
+                            LogMissingError(apiResponse);
+                            return new ApiRequestException(apiResponse.Message, 403)
+                            {
+                                Parameters = apiResponse.Parameters
+                            };
+                    }
+                default:
+                    LogMissingError(apiResponse);
+                    return new ApiRequestException(apiResponse.Message, apiResponse.Code)
+                    {
+                        Parameters = apiResponse.Parameters
+                    };
+            }
+        }
+
+        private static void LogMissingError<T>(ApiResponse<T> apiResponse)
```

Comment:
What is this supposed to do?
We could add some `Debug.WriteLine()` statements here.